api/stats.js

```diff
@@ -103,21 +119,40 @@ export class Stats {
         return stats;
     };
 
-    _sendStatsChannel = async (stats, title) => {
-        if (!stats || !Object.keys(stats).length) {
+    _sendStatsChannel = async (serverStats, title) => {
+        if (!serverStats || !Object.keys(serverStats).length) {
             // nothing to update
             return;
         }
-        const postsValue = Object.keys(stats).filter((name) => !config.BOTS.includes(name))
-            .map((username) => `**${username}** **(** ${stats[username].posts} **|** ${stats[username].words} **|** ${stats[username].letters}** )**`)
+        let inactive = [];
+        let active = [];
+        for (let user in serverStats) {
+            if (config.BOTS.has(user)) {
+                continue;
+            }
+
+            if (serverStats[user].posts) {
+                active.push([user, serverStats[user]]);
+            } else {
+                inactive.push(user);
+            }
+        }
+        active.sort((a, b) => b[1].posts - a[1].posts);
+        inactive.sort((a, b) => a - b);
```

Comment:
Sorting alphabetically, however, once we start counting days of inactivity we sort by that. https://github.com/kajahno/elprimobot/issues/65